pyproject.toml

```diff
@@ -0,0 +1,113 @@
+[build-system]
+build-backend = "setuptools.build_meta:__legacy__"
+requires = [
+    "setuptools>=64.0.0",
+    "tomli  ; python_version < '3.11'",
```

Comment:
Support for TOML is directly integrated into Python 3.11.0. So no need for this dependency there. Hence this is a backport included for support of older Python versions.

This is just the format for the file. Please see [this example]( https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use )

Reply:
Ah ok, this is [PEP-508](https://peps.python.org/pep-0508/), which is just inconsistent like this for presumably reasons.

Reply:
The different types of quoting is just a Python trick to avoid needing to escape the quotes with `\`. Python views these two strings `"abc"` and `'abc'` as equivalent.